### ISSUE TITLE
Link to OMIM

### ISSUE THREAD

**Issue by pnrobinson** [TheJacksonLaboratory/hpo-web#239]
Hi @iimpulse and @jmcmurry 

Julie, in case you have some time to give design advice:

I was thinking we could start to link to MONDO ids that we have mapped.
Also, it would be good to link from the top of the entry to the OMIM page. Previously, all entries had links from the HPO term to the OMIM page, but that is no longer the case for many hand-curated entries, e.g.,
https://hpo.jax.org/app/browse/disease/OMIM:619503

Another thing is that we have excluded annotations, i.e., those with zero frequency. Maybe we should show all excluded annotations in a separate panel?

Finally, it would be great to start putting the MAxO annotations. There are two kinds of annotations -- diagnostic ones should be shown for HPO terms, like the loinc2HPO annotation here https://hpo.jax.org/app/browse/term/HP:0002902

The second kind comes from POET and should be shown for the diseases on the disease pages. Could we start to think about this?

**Comment by jmcmurry** [comment]
was thinking we could start to link to MONDO ids that we have mapped.

 -- yes that would be great

Also, it would be good to link from the top of the entry to the OMIM page. Previously, all entries had links from the HPO term to the OMIM page, but that is no longer the case for many hand-curated entries, e.g.,
https://hpo.jax.org/app/browse/disease/OMIM:619503

-- that is fine, just use the link out icon so it is clear the user is leaving the application

Another thing is that we have excluded annotations, i.e., those with zero frequency. Maybe we should show all excluded annotations in a separate panel?

-- Yes an all excluded panel would be great, possibly with a modification to the positively-asserted tables such as (see 2 excluded head and neck phenotpyes)

Finally, it would be great to start putting the MAxO annotations. There are two kinds of annotations -- diagnostic ones should be shown for HPO terms, like the loinc2HPO annotation here https://hpo.jax.org/app/browse/term/HP:0002902

-- Yes, that would be great. 

The second kind comes from POET and should be shown for the diseases on the disease pages. Could we start to think about this?

-- Definitely. Do the diagnostic ones not originate in POET?
